packages/integration-tests/src/tests/api/experience-api/sign-in-interaction/enterprise-sso.test.ts

```diff
@@ -98,6 +98,19 @@ describe('enterprise sso sign-in and sign-up', () => {
       expect(tokenSecret?.metadata.scope).toBe('openid profile email');
     }
 
+    // Should delete the token set when the connector token storage is disabled
+    if (isDevFeaturesEnabled) {
+      await ssoConnectorApi.update(ssoConnectorApi.firstConnectorId!, {
```

Comment:
better not to use xxx!, instead we can throw when firstConnectorId is not provided, although this case should not happen, we can add comments to show this.

Reply:
updated